# HG changeset patch
# User Alexander Dreyer <adreyer@gmx.de>
# Date 1343046213 -7200
# Node ID bba2f09adf6d9961ca1d7054495f1f39454542c2
# Parent  72f94f6ccbc1fb2d32ca8ea3f917ed4721f15247
Fix: more stable (stripped!) access to $PYTHON in env

diff -r 72f94f6ccbc1fb2d32ca8ea3f917ed4721f15247 -r bba2f09adf6d9961ca1d7054495f1f39454542c2 SConstruct
--- a/SConstruct	Mon Jul 16 15:23:22 2012 +0200
+++ b/SConstruct	Mon Jul 23 14:23:33 2012 +0200
@@ -216,7 +216,7 @@
 
 def _moduleflags(env):
     if env['PLATFORM']=="darwin":
-        python_absolute = shell_output("which", env["PYTHON"])
+        python_absolute = shell_output("which", env.subst("$PYTHON"))
         return ["-fvisibility=hidden", "-bundle_loader", python_absolute]
 
     return []
